src/vs/editor/browser/services/codeEditorServiceImpl.ts

```diff
@@ -310,7 +310,7 @@ class DecorationRenderHelper {
 				cssTextArr.push(strings.format(this._CSS_MAP.contentIconPath, opts.contentIconPath.toString(true).replace(/'/g, '%27')));
```

Comment:
This was actually a VSCode bug that I discovered. You couldn't have after/before text decorators with a `'` in them. `_CSS_MAP.contentText` is defined as `contentText: 'content:\'{0}\';',` . This means the escaping of `"` actually did nothing. It should have been escaping single quotes `'`.  I'm fairly certain that this doesn't have side effects because its only escaping the contentText property and that's the only usage of it. 

Without this fix,python string values won't show up. Since python strings show up as `'cat bat mat'` in the watches view.